back-office/orders/orders/Order Add/Add new Order.md

```diff
@@ -206,22 +206,30 @@ A soft "deleted" address can be used in the cart / order.
 
 ## V. Payment and status configuration
 
+
 ### Order message
+
 The merchant can add a message to the order. After the creation, the message is displayed in the "Messages" panel of the order detail page as a private message: with a dark background and the specific icon for the sender pic.
 
 Invalid characters are: <>{}
 When you save, if your order message contains an invalid character, an error notification should be displayed “The order message given is invalid”.
 
+
 ### Send an email to the customer with the link to process the payment
 
 This button allows to send an email to the customer with the link to process the payment
 It's not related to the payment method selected.
-The link opens the front-office with the cart corresponding to the order created.
+
+Next to this button is a hyperlink to allow the user to proceed to the payment directly in the front office. Clicking this hyperlink opens a new tab that redirects the user to the payment step of the checkout process for this order - the user does not need to fill all the personal information, addresses, or shipping method but he must be able to navigate through those previous steps before going back to the payment.
+
+Processing the payment of this order through the checkout in the front office immediately creates the order in the back office. As a consequence, the user is not creating a new order anymore and must go in the Orders > Orders section of his/her back office to reach this order.
```

Comment:
```suggestion
Processing the payment through the checkout in the front office automatically creates the order and adds it to the orders listing in the back office. As a consequence, the merchant does not have to complete the ongoing creation of the new order.

If the payment is not processed right away, the merchant must keep on completing the creation form in the back office and click 'Create new order' to add it to the orders listing.
```

Reply:
@Robin-Fischer-PS

Reply:
Thanks for the edit, it clears things up !